docker/snaptools/makeCompliantBAM.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import pysam
+import argparse
+import sys
+import os
+
+def main():
+    parser = argparse.ArgumentParser(description="Make a BAM file with cellular barcodes in the "
```

Comment:
I'd suggest a simpler description: 

```python
    parser = argparse.ArgumentParser(description="Move barcodes from the read name to the CB tag")
```

general comment: you don't need the plus between the strings, python will concatenate them across lines automatically.